spec/classes/init_spec.rb

```diff
@@ -293,6 +296,7 @@
               value: '10000',
             )
           }
+          it { is_expected.to contain__loginctl_user('foo').with(linger: 'enabled') }
```

Comment:
Looks like I missed another thing in my suggestion:
```suggestion
          it { is_expected.to contain_loginctl_user('foo').with(linger: 'enabled') }
```